apps/opencs/model/world/columnimp.hpp

```diff
@@ -60,6 +63,20 @@ namespace CSMWorld
         }
     };
 
+    template<>
+    inline QVariant StringIdColumn<Land>::get(const Record<Land>& record) const
+    {
+        const Land& land = record.get();
+        return QString(Land::createUniqueRecordId(land.mX, land.mY).c_str());
```

Comment:
IIRC that is not the correct way to convert between std::string and QString (at least not for Qt4). There should be plenty of examples in the code on how to convert properly.

Reply:
[Qt5](http://doc.qt.io/qt-5/qstring.html#QString-7)
[Qt4](http://doc.qt.io/qt-4.8/qstring.html#QString-8)
I can use the another method in case we plan on defining QT_NO_CAST_FROM_ASCII at some point.

Reply:
Just looked it up. Here is an example from our code base:

> QString::fromUtf8 (mText.c_str())